docker-vertica/Dockerfile

```diff
@@ -87,66 +87,72 @@ RUN set -x \
   && chmod 777 -R /opt/vertica
 
 ##############################################################################################
-FROM centos:centos${OS_VERSION}
+FROM ubuntu:${BASE_OS_VERSION}
 
 ARG DBADMIN_GID=5000
 ARG DBADMIN_UID=5000
 
 COPY --from=builder /opt/vertica /opt/vertica
 COPY --from=builder /home/dbadmin /home/dbadmin
 COPY --from=builder /root/.ssh /root/.ssh
-COPY --from=builder /etc/ssh /etc/ssh
 COPY --from=builder /var/spool/cron /var/spool/cron
 
 ENV LANG en_US.utf8
 ENV TZ UTC
 ENV PATH "$PATH:/opt/vertica/bin:/opt/vertica/sbin"
+ENV DEBIAN_FRONTEND noninteractive
 
 ADD ./packages/init.d.functions /etc/rc.d/init.d/functions
 
+
+# make the "en_US.UTF-8" locale so vertica will be utf-8 enabled by default
+RUN apt-get update && apt-get install -y locales && rm -rf /var/lib/apt/lists/* \
```

Comment:
Can we combine this RUN command with the one below?  That will save a layer in the docker image.